.github/workflows/test.yml

```diff
@@ -6,4 +6,17 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - run: echo "TODO Run automated tests"
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 14
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/maven-jar/pom.xml') }}
```

Comment:
Can we create two test runners, one for maven-jar and another for maven-services?